acceptance/setup/common/035_StartPuppetServer.rb

```diff
@@ -1,2 +1,2 @@
 # Start puppetserver - this is a separate pre-suite block because many pre-suites don't actually want/need the server running
-on(master, puppet('resource', 'service', 'puppetserver', "ensure=running"))
+on(master, puppet('resource', 'service', 'puppetserver', 'ensure=running', 'enable=true'))
```

Comment:
It seems correct to have the service enabled, but I'm curious what necessitated this change - did we have instances of the master node restarting? I'll message you separately from this PR, as I've no objection to merging this, just curious.